.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  parser:  '@typescript-eslint/parser',
```

Comment:
move this to server/ .eslintrc

Reply:
Don't we need this for `index.js` too?

Reply:
you're right... maybe move app and index to inside server/?